airflow/scripts/git_scripts/aggregate_repo_activity.py

```diff
@@ -0,0 +1,85 @@
+
+import requests
```

Comment:
Please add a requirement.txt file.

Reply:
In what directory should I add the requirements.txt file

Reply:
The same directory as your script so we can create venv to your script and include the requirement.txt.
Change the name to "git_collector" instead of "git_scripts" because we may create more git scripts on airflow.
Path : 
airflow/scripts/git_collector/requirement.txt